src/main/java/org/maxgamer/rs/model/interact/JavaScriptInteractHandler.java

```diff
@@ -118,7 +118,7 @@ public void javascript(Mob source, Interactable target, String option) throws Su
         for (File f : files) {
             if(!f.exists()) continue;
 
-            String module = f.getAbsolutePath().substring(INTERACTION_FOLDER.getAbsoluteFile().getParentFile().getAbsolutePath().length() + 1);
+            String module = f.toURI().toString();
```

Comment:
I think I was trying to make sure the URI was relative instead of absolute here. 

All of the URL's should be relative though, so seems fine